lib/kolekti/runner.rb

```diff
@@ -40,7 +40,10 @@ def clean_output
 
     def find_collectors
       collectors = wanted_metric_configurations.map do |code, metric_configuration|
-        found_collector = Kolekti.collectors.find { |collector| collector.supported_metrics.include? code }
+        collector_name = metric_configuration.metric.metric_collector_name
+        found_collector = Kolekti.collectors.find { |collector|
```

Comment:
What do you think about using `find do |collector| ... end` since you are alre using multilines?